<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Label_user">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>121</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_passwd">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>101</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_reminder">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提示：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_user">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>321</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_login">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>120</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_register">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>120</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_exit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>120</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_passwd">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>320</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
